draft-thomson-webpush-protocol.xml

```diff
@@ -364,7 +407,71 @@ Location: https://push.example.net/d/qDIYHNcfAIPP_5ITvURr-d6BGtYnTRnk
         Push services MUST NOT generate a 413 status code in responses to an
         entity body that is 4k (4096 bytes) or less in size.
       </t>
+      <section anchor="request_receipt" title="Receiving Push Message Receipts">
+        <t>
+          An application server can use the Receipt header field to request a
+          confirmation from the push server when a push message is delivered and
+          acknowledged by the user agent. The Receipt header field value is the
+          "receipt" URI created when the application server subscribed to
+          receipts.
+        </t>
+        <figure>
+          <artwork type="abnf">
+            <![CDATA[
+Receipt = URI-reference
```

Comment:
RFC 7230 only references 3986 by reference, so either is fine (we already cite 7230, so maybe that's easier ;)